pick.py

```diff
@@ -7,13 +7,12 @@
 from utils import printstr, writestr, pbcopy
 from table_view import TableView
 
-# TODO: Scroll for big tables (curses pad)
-
 ss = inspect.cleandoc
 
 parser = argparse.ArgumentParser(prog='pick',
                                  description="""Interactively lets the user pick a cell in a table given to the
-                                                standard input. The delimiter for the columns may be chosen, whereas
+                                                standard input. The delimiter for the
+                                                columns may be chosen, whereas
```

Comment:
It seems your editor is breaking lines at 80 char limit, we can agree to that but not on this PR, please relax it to 120 chars and then we change everything to 80 in the future if it's the case